tests/playwright-test/babel.spec.ts

```diff
@@ -36,3 +36,43 @@ test('should succeed', async ({ runInlineTest }) => {
   expect(result.passed).toBe(1);
   expect(result.failed).toBe(0);
 });
+
+test('should treat enums equally', async ({ runInlineTest }) => {
+  test.fail(true, 'https://github.com/microsoft/playwright/issues/13265');
```

Comment:
We started to do it differently: https://github.com/microsoft/playwright/blob/02cac8a0668c59078eb35029ed88c9f88e139428/tests/library/browsercontext-page-event.spec.ts#L189